conjure-java-undertow-runtime/src/test/java/com/palantir/conjure/java/undertow/runtime/ConjureHandlerBuilderTest.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * (c) Copyright 2019 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.conjure.java.undertow.runtime;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import com.google.common.collect.ImmutableList;
+import com.palantir.conjure.java.undertow.lib.Endpoint;
+import com.palantir.logsafe.SafeArg;
+import com.palantir.logsafe.SafeLoggable;
+import com.palantir.logsafe.exceptions.SafeIllegalArgumentException;
+import io.undertow.server.HttpHandler;
+import io.undertow.util.HttpString;
+import io.undertow.util.Methods;
+import org.junit.Test;
+
+public class ConjureHandlerBuilderTest {
+
+    @Test
+    public void addOverlappingEndpoints() {
+        ConjureHandler.Builder builder = ConjureHandler.builder()
+                .endpoints(buildEndpoint(Methods.GET, "/foo", "serviceName1", "bar"))
+                .endpoints(buildEndpoint(Methods.GET, "/foo", "serviceName1", "bar2"))
+                .endpoints(buildEndpoint(Methods.GET, "/foo", "serviceName2", "bar"))
+                .endpoints(buildEndpoint(Methods.POST, "/foo", "serviceName1", "bar"))
+                .endpoints(buildEndpoint(Methods.POST, "/foo", "serviceName2", "bar"))
+                .endpoints(buildEndpoint(Methods.GET, "/foo2", "serviceName1", "bar"));
```

Comment:
Can we add endpoints with paths `/foo/{a}` and `/foo/{b}`?

Reply:
done